src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -43,7 +45,7 @@
      * @see #CLASS_DEF
      * @see #INTERFACE_DEF
      **/
-    public static final int EOF = GeneratedJavaTokenTypes.EOF;
+    public static final int EOF = Recognizer.EOF;
```

Comment:
This must be `Recognizer` and not `JavaParser`.

From https://teamcity.jetbrains.com/viewLog.html?buildId=3543062&tab=Inspection&buildTypeId=Checkstyle_IdeaInspectionsPullRequest:

```
46: EOF Static field EOF declared in class 'org.antlr.v4.runtime.Recognizer' but referenced via subclass 'com.puppycrawl.tools.checkstyle.grammar.java.JavaLexer' 

```